assets/cubyz/biomes/desert.zig.zon

```diff
@@ -25,5 +25,28 @@
 			.height_variation = 3,
 		},
 	},
+	.transitionBiomes = .{
+		.{
+			.id = "cubyz:trans/sandy_coast",
+			.chance = 1,
+			.width = 1,
+			.keepOriginalTerrain = 0.5,
+			.properties = .{.ocean, .wet},
+		},
+		.{
+			.id = "cubyz:trans/drylands",
```

Comment:
I'm also not sold on this biome. It does seem to cause some trouble:
![Screenshot at 2025-01-09 18-08-48](https://github.com/user-attachments/assets/7e705309-2b53-4cda-bbc9-75957d433047)
![Screenshot at 2025-01-09 18-09-12](https://github.com/user-attachments/assets/b3fa443d-7457-4239-85ba-54f8b7395e9c)
![Screenshot at 2025-01-09 18-10-31](https://github.com/user-attachments/assets/184110af-2af8-48bd-8fd1-6aaf69a02898)

Reply:
Then how about you exclude these for now from this PR, and add them in a future PR, after all the other biome changes have been made?

Reply:
Could you also exclude the related files from this PR, I don't want to have to review these just now.